devimint/src/federation.rs

```diff
@@ -399,7 +399,7 @@ impl Federation {
         let start_time = Instant::now();
         debug!(target: LOG_DEVIMINT, "Awaiting LN gateways registration");
         let fedimint_cli_version = crate::util::FedimintCli::version_or_default().await;
-        let command = if VersionReq::parse("<0.3.0-alpha")?.matches(&fedimint_cli_version) {
+        let command = if fedimint_cli_version < Version::parse("0.3.0-alpha")? {
```

Comment:
Just a thought: do we want to put all of our version constants in a central place (maybe a new file?). Might help with reasoning about which versions have changes between them.

Reply:
https://github.com/fedimint/fedimint/pull/4828